### PR TITLE
Made the instructions clearer

### PR DESCRIPTION
**Pull request by letzya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,11 @@ This enables multicluster, multi Data-Centre API management from a single Dashbo
 The Tyk owned MDCB registry is private and requires adding users to our organisation which you then define as a secret when pulling the MDCB image. Please contact your account manager to arrange this.
 
 ## Install Tyk Hybrid Gateways (This can be used either for Multi-Cloud Gateways or MDCB slaves)
-To install, *first modify the `values_hybrid.yaml` file to add redis details, your RPC key, API key and Dashboard URL*:
+To install, first modify `values_hybrid.yaml` file as follows:
+1. Add redis passwordi in redis.pass. It's the value of $REDIS_PASSWORD (The host should be "tyk-redis-master.tyk-ingress.svc.cluster.local" if you used the tyk-ingress as the namespace.
```
Comment (excieve) [inline README.md]:
```suggestion
1. Add redis password in `redis.pass` value. It's the value of `$REDIS_PASSWORD` environment variable (the host should be "tyk-redis-master.tyk-ingress.svc.cluster.local" if you used the tyk-ingress as the namespace.
```

File: README.md [hunk 2]
```diff
@@ -59,7 +59,11 @@ This enables multicluster, multi Data-Centre API management from a single Dashbo
...
+2. Add your RPC key (i.e. ORGID) in tyk_k8s.org_id
```
Comment (excieve) [inline README.md]:
```suggestion
2. Add your RPC key (i.e. ORGID) in `tyk_k8s.org_id` value
```

File: README.md [hunk 3]
```diff
@@ -59,7 +59,11 @@ This enables multicluster, multi Data-Centre API management from a single Dashbo
...
+3. Add your API KEY in tyk_k8s.dash_key (Could be the api key of any dashboard user but better to have a dedicated one)
```
Comment (excieve) [inline README.md]:
```suggestion
3. Add your API key in `tyk_k8s.dash_key` value (could be the API key of any dashboard user but better to have a dedicated one) 
```

File: README.md [hunk 4]
```diff
@@ -59,7 +59,11 @@ This enables multicluster, multi Data-Centre API management from a single Dashbo
 The Tyk owned MDCB registry is private and requires adding users to our organisation which you then define as a secret when pulling the MDCB image. Please contact your account manager to arrange this.
 
 ## Install Tyk Hybrid Gateways (This can be used either for Multi-Cloud Gateways or MDCB slaves)
-To install, *first modify the `values_hybrid.yaml` file to add redis details, your RPC key, API key and Dashboard URL*:
+To install, first modify `values_hybrid.yaml` file as follows:
+1. Add redis passwordi in redis.pass. It's the value of $REDIS_PASSWORD (The host should be "tyk-redis-master.tyk-ingress.svc.cluster.local" if you used the tyk-ingress as the namespace.
+2. Add your RPC key (i.e. ORGID) in tyk_k8s.org_id
+3. Add your API KEY in tyk_k8s.dash_key (Could be the api key of any dashboard user but better to have a dedicated one) 
+4. Add your dashboard url in tyk_k8s.dash_url. If it's a multi cloud account the value *https://admin.cloud.tyk.io* is already set for you.
```
Comment (excieve) [inline README.md]:
```suggestion
4. Add your dashboard URL in `tyk_k8s.dash_url` value. If it's a Tyk SaaS account the value *https://admin.cloud.tyk.io* is already set for you.
```

Comment (letzya) [reply README.md]: Why `Tyk SaaS` and not `Tyk Multi Cloud`?

File: README.md (hunk 2 above)
Comment (letzya) [reply README.md]: @excieve  - I removed **(i.e. ORGID)**

File: README.md (hunk 4 above)
Comment (excieve) [reply README.md]: We don't have Multi Cloud anymore. All SaaS accounts are hybrid.

**Comment by letzya** [comment]
@excieve can we merge, please?

**Comment by excieve** [comment]
@letzya Looks like two suggestions are still outstanding.

**Pull request merged**